cmd/crc/cmd/start.go

```diff
@@ -133,3 +137,20 @@ func getPullSecretFileContent() (string, error) {
 
 	return pullsecret, nil
 }
+
+func checkIfNewVersionAvailable(noUpdateCheck bool) {
+	if noUpdateCheck {
+		return
+	} else {
```

Comment:
@anjannath you don't need `else` here if you already returning it from the `if` block.

Reply:
Right, i'll push another update, Thanks!